src/sentry/sentry_apps/services/app/service.py

```diff
@@ -66,6 +67,27 @@ def get_installed_for_organization(
         self,
         *,
         organization_id: int,
+    ) -> list[RpcSentryAppInstallation]:
+        # Deprecated use installations_for_organization instead.
+        pass
+
+    def installations_for_organization(
+        self, *, organization_id: int
+    ) -> list[RpcSentryAppInstallation]:
+        """
+        Get a list of installations for an organization_id
+
+        This is a cached wrapper around get_installations_for_organization
+        """
+        if in_random_rollout("app_service.installations_for_org.cached"):
+            return get_installations_for_organization(organization_id)
+        else:
+            return self.get_installed_for_organization(organization_id=organization_id)
```

Comment:
Once these changes are fully deployed, I plan on using this option to shift load onto caching and hope to observe changes in RPC throughput and post_process task duration.